packages/css-library/src/stylesheets/_elements.scss

```diff
@@ -1,148 +1 @@
-/*! This is intentionally left empty for now */
```

Comment:
I'm removing the content of this stylesheet because it's essentially just a duplicate of `_va.scss` at this point. We will probably just want to use that stylesheet when we start migrating the Formation links to CSS Library.